tauri-app/src-tauri/src/main.rs

```diff
@@ -3,14 +3,16 @@
 
 mod commands;
 use commands::vault::{vault_detail, vaults_list};
-use commands::wallet::get_address_from_ledger;
+// use commands::wallet::get_address_from_ledger;
```

Comment:
commented code

Reply:
commented because it was part of the code, but was throwing errors, so I commented it, along with the wallet.rs

Reply:
It's throwing errors because you've pinned `alloy-ethers-typecast` to branch `2024-01-24-fix-ethers-dep` which is outdated